### PR TITLE
feat(api-client): Add TeamMemberUpdateData

### PR DESCRIPTION
**Pull request by ffflorian**
## Pull Request Checklist

- [ ] My code is covered by tests
- [x] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/api-client/src/team/data/TeamMemberUpdateData.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {PermissionsData} from '../member/PermissionsData';
+
+export interface TeamMemberUpdateData {
```
Comment (Yserz) [inline packages/api-client/src/team/data/TeamMemberUpdateData.ts:22]: Does it also contain `created_by` & `created_at` like in `MemberData`?

Comment (ffflorian) [reply packages/api-client/src/team/data/TeamMemberUpdateData.ts:22]: No:

Screenshot

Comment (Yserz) [reply packages/api-client/src/team/data/TeamMemberUpdateData.ts:22]: It's optional so does this user have these values?

Comment (ffflorian) [reply packages/api-client/src/team/data/TeamMemberUpdateData.ts:22]: Mhh it's possible that it just returns `MemberData`..

Comment (ffflorian) [reply packages/api-client/src/team/data/TeamMemberUpdateData.ts:22]: AFAIC the server returns only `user` and `permissions`: https://github.com/wireapp/wire-server/blob/fffbd95254cbd273646242af23feb30af3ea797b/libs/galley-types/src/Galley/Types/Teams.hs#L908

Comment (Yserz) [reply packages/api-client/src/team/data/TeamMemberUpdateData.ts:22]: Alright, thx for digging :)

**Pull request merged**